.github/workflows/build.yaml

```diff
@@ -109,12 +109,16 @@ jobs:
           kubectl describe node
       - name: Run integration tests
         run: |
+          kubectl apply -f deploy/static/01-starboard-operator.ns.yaml \
```

Comment:
We're running the operator as cluster admin out of cluster to generate test coverage report. However, scan jobs are run with `starboard-operator` service account. Therefore, proper RBAC config is required before we run the test suite.